Client/core/CSettings.cpp

```diff
@@ -1006,66 +1006,36 @@ void CSettings::CreateGUI ( void )
         // Layout tab
         CGUITab* pLayoutTab = pChatOptionsPanel->CreateTab ( _("Layout") );
 
-        pLabel = reinterpret_cast < CGUILabel* > ( pManager->CreateLabel ( pLayoutTab, "X:" ) );
-        pLabel->SetPosition ( CVector2D ( 10.0f, 10.0f ) );
-        pLabel->GetPosition ( vecTemp );
-        pLabel->AutoSize ( );
-        pLabel->SetAlwaysOnTop ( true );
-
-        m_pChatPosX = reinterpret_cast < CGUIEdit* > ( pManager->CreateEdit ( pLayoutTab, "") );
-        m_pChatPosX->SetPosition ( CVector2D ( vecTemp.fX + 12.0f, vecTemp.fY - 2.0f ) );
-        m_pChatPosX->GetPosition ( vecTemp );
-        m_pChatPosX->SetSize ( CVector2D ( 70.0f, 24.0f ) );
-        m_pChatPosX->SetAlwaysOnTop ( true );
-
-        pLabel = reinterpret_cast < CGUILabel* > ( pManager->CreateLabel ( pLayoutTab, "Y:" ) );
-        pLabel->SetPosition ( CVector2D ( vecTemp.fX + 75.0f, vecTemp.fY + 2.0f ) );
-        pLabel->GetPosition ( vecTemp );
-        pLabel->AutoSize ( );
-        pLabel->SetAlwaysOnTop ( true );
-
-        m_pChatPosY = reinterpret_cast < CGUIEdit* > ( pManager->CreateEdit ( pLayoutTab, "") );
-        m_pChatPosY->SetPosition ( CVector2D ( vecTemp.fX + 12.0f, vecTemp.fY - 2.0f ) );
-        m_pChatPosY->SetSize ( CVector2D ( 70.0f, 24.0f ) );
-        m_pChatPosY->SetAlwaysOnTop ( true );
-
         pLabel = reinterpret_cast < CGUILabel* > ( pManager->CreateLabel ( pLayoutTab, _("Lines:") ) );
-        pLabel->SetPosition ( CVector2D ( 10.0f, vecTemp.fY + fLineSizeY + fLineGapY ) );
+        pLabel->SetPosition ( CVector2D ( 10.0f, 10.0f ) );
         pLabel->GetPosition ( vecTemp );
         pLabel->AutoSize ( );
-        pLabel->SetAlwaysOnTop ( true );

```

Comment:
Some time ago there was no tab panel for "Layout" and "Fading", which I introduced at one point. Before that there was some kind of panel in the background for unknown reasons and to keep the GUI elements above the panel they needed the "AlwaysOnTop" property. I removed these lines in the code now since I realized it in the last days.